smartsim/_core/control/controller.py

```diff
@@ -135,6 +135,10 @@ def start(
             # it may be called seperately
             self.poll(5, True, kill_on_interrupt=kill_on_interrupt)
 
+    @property
```

Comment:
Should we put doc strings for this method (I noticed you put it for the property method in manifest.py)?

Reply:
Added doc string